data/mods/innawood/items/lighting.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "id": "oil_lamp_on",
+    "copy-from": "oil_lamp",
+    "type": "TOOL",
+    "name": { "str": "oil lamp" },
+    "description": "This is an oil fueled lamp.  It does not provide much light, but it lasts a long time.  It is turned on.  Use it to turn it off.",
+    "turns_per_charge": 240,
+    "revert_to": "oil_lamp",
+    "use_action": [
+      {
+        "ammo_scale": 0,
+        "target": "oil_lamp",
+        "msg": "The lantern is extinguished.",
+        "menu_text": "Turn off",
+        "type": "transform"
+      },
+      { "type": "firestarter", "moves": 100 }
+    ],
+    "flags": [ "LIGHT_30", "TRADER_AVOID", "FIRE", "ALLOWS_REMOTE_USE", "WATER_EXTINGUISH", "FIRESTARTER" ]
+  },
+  {
+    "id": "oil_lamp_clay",
```

Comment:
Are you missing a recipe for this item? Also I believe ancient lamp oil would be acceptable in mainline as long as you cite your sources?

Reply:
I believe that is part of rebalancing the original oil lamp to have a different light level.

Reply:
Yes thats right, its just the original clay oil lamp but with more light and the "new" lamp oil instead of kerosene. The reintroduced natural lamp oil is the only actually new item.

Still, it might make sense to tweak mainline clay oil lamps in this way to make it more different from the modern "oil lamp". If you craft a clay oil lamp, it would make sense that you could fill it with different oils than kerosene. But maybe thats overly complicating things. I think the torchlight nerf should be in mainline though.